src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -719,6 +719,11 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 _cursorTimer = std::nullopt;
             }
 
+            // Set up a hover timer, but don't start it yet
+            int hoverTime = 500;

```

Comment:
this could probably be a static constexpr at the top of this file

Reply:
Not needed anymore!